docs/titan-basics/other-parts-of-the-touch-screen.md

```diff
@@ -75,7 +75,7 @@ meet in the middle.
 
 ![Sapphire Tools Menu](/docs/images/Sapphire-Tools-Menu.png)
 
-On Diamond 9 the toolbar is replaced by a small <Keys.ContextKey>Tools</Keys.ContextKey> button at the top
+On Diamond 9 / Diamond 7 the toolbar is replaced by a small <Keys.ContextKey>Tools</Keys.ContextKey> button at the top
```

Comment:
On the D7 it is on the top left of the left hand screen on a 2 screen or middle screen on a 3 screen.